util/http/errors.go

```diff
@@ -0,0 +1,19 @@
+/*
+   Copyright The Soci Snapshotter Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package http
+
+const ECRTokenExpiredResponse = "Your authorization token has expired. Reauthenticate and try again."
```

Comment:
Wasn't sure where to put this. 

I do think this is a good place for storing general errors and it should enforce us to standardize some of our errors in resolver.go.